common/lib/connection_plugin_chain_builder.ts

```diff
@@ -37,6 +37,8 @@ import { ConnectionProviderManager } from "./connection_provider_manager";
 import { DeveloperConnectionPluginFactory } from "./plugins/dev/developer_connection_plugin_factory";
 import { ConnectionPluginFactory } from "./plugin_factory";
 import { LimitlessConnectionPluginFactory } from "./plugins/limitless/limitless_connection_plugin_factory";
+import { FastestResponseStrategyPlugin } from "./plugins/strategy/fastest_response/fastest_response_strategy_plugin";
```

Comment:
I don't believe the plugin needs to be imported here

Reply:
thanks